crates/bevy_feathers/src/controls/slider.rs

```diff
@@ -0,0 +1,214 @@
+use core::f32::consts::PI;
+
+use bevy_app::{Plugin, PreUpdate};
+use bevy_color::Color;
+use bevy_core_widgets::{CoreSlider, SliderRange, SliderValue, TrackClick};
+use bevy_ecs::{
+    bundle::Bundle,
+    children,
+    component::Component,
+    entity::Entity,
+    hierarchy::Children,
+    lifecycle::RemovedComponents,
+    query::{Added, Changed, Has, Or, Spawned, With},
+    schedule::IntoScheduleConfigs,
+    spawn::SpawnRelated,
+    system::{In, Query, Res, SystemId},
+};
+use bevy_input_focus::tab_navigation::TabIndex;
+use bevy_picking::PickingSystems;
+use bevy_ui::{
+    widget::Text, AlignItems, BackgroundGradient, ColorStop, Display, FlexDirection, Gradient,
+    InteractionDisabled, JustifyContent, LinearGradient, Node, UiRect, Val,
+};
+use bevy_winit::cursor::CursorIcon;
+
+use crate::{
+    font_styles::InheritableFont,
+    handle_or_path::HandleOrPath,
+    theme::{
+        self,
+        corners::RoundedCorners,
+        fonts, size,
+        tokens::{SLIDER_BAR, SLIDER_BAR_DISABLED, SLIDER_BG},
+        ThemeFontColor, UiTheme, UseTheme,
+    },
+};
+
+/// Slider template properties, passed to [`slider`] function.
+// TODO: These should all be replaced with reactive signals.
```

Comment:
Not convinced that this TODO is net helpful: that work is fairly blocked, so it's mostly just going to be noise.